pkg/controllers/controllers.go

```diff
@@ -42,7 +43,8 @@ import (
 	"github.com/aws/karpenter-provider-aws/pkg/providers/amifamily"
 	"github.com/aws/karpenter-provider-aws/pkg/providers/instance"
 	"github.com/aws/karpenter-provider-aws/pkg/providers/instanceprofile"
-	"github.com/aws/karpenter-provider-aws/pkg/providers/pricing"
+	providersinstancetype "github.com/aws/karpenter-provider-aws/pkg/providers/instancetype"
```

Comment:
We really only need to alias one of ["controllers", "providers"]. Aliasing controllers seems reasonable to me, so I'd just do that one